singer/src/main/java/com/pinterest/singer/utils/LogConfigUtils.java

```diff
@@ -950,9 +950,8 @@ private static LogStreamReaderConfig parseLogStreamReaderConfig(AbstractConfigur
   }
 
   private static ThriftReaderConfig parseThriftReaderConfig(AbstractConfiguration thriftReaderConfiguration) {
-    thriftReaderConfiguration.setThrowExceptionOnMissing(true);
-    int readerBufferSize = thriftReaderConfiguration.getInt("readerBufferSize");
-    int maxMessageSize = thriftReaderConfiguration.getInt("maxMessageSize");
+    int readerBufferSize = thriftReaderConfiguration.getInt("readerBufferSize", 10240);
+    int maxMessageSize = thriftReaderConfiguration.getInt("maxMessageSize", 100000);
```

Comment:
Nit: Defining defaults like these in one place (e.g. using constants) and referencing them here would make finding those defaults easier in the future.

Reply:
Good catch, was too focused on the bug fix so missed this. Fixed now.